engine/runtime-instrument-common/src/main/scala/org/enso/interpreter/instrument/job/UpsertVisualizationJob.scala

```diff
@@ -243,7 +249,7 @@ object UpsertVisualizationJob {
                 argumentExpressions
               ) =>
             val callback = ctx.executionService.prepareFunctionCall(
-              module,
+              expressionModule,
```

Comment:
If there are two modules now, `modue` - for evaluating arguments, and the `expressionModule` for evaluating the expression, you should use the second one when evaluating the text expression ^
```scala
          case Api.VisualizationExpression.Text(_, expression) =>
            val callback = ctx.executionService.evaluateExpression(
              module,
              expression
            )
```
https://github.com/enso-org/enso/pull/7115/files#diff-3dbde2d45b1e4c5cf9dabfd23c1e7d89896ccac7ee7b8b694d65c1d6fd950037R240-R244

Reply:
Is this what you are requesting: 38c486e110da8c3fd5b0b5ee81494a3e71f03f97 ?
```diff
enso$ git show
commit 38c486e110da8c3fd5b0b5ee81494a3e71f03f97 (HEAD -> wip/jtulach/UnresolvedSymbol_6955)
Author: Jaroslav Tulach <jaroslav.tulach@enso.org>
Date:   Tue Jun 27 15:51:08 2023 +0200

    Evaluate expression in expressionModule

diff --git engine/runtime-instrument-common/src/main/scala/org/enso/interpreter/instrument/job/UpsertVisualizationJob.scala engine/runtime-instrument-common/src/main/scala/org/enso/interpreter/instrument/job/UpsertVisualizationJob.scala
index 9dfabf21b8..1d87cdc93e 100644
--- engine/runtime-instrument-common/src/main/scala/org/enso/interpreter/instrument/job/UpsertVisualizationJob.scala
+++ engine/runtime-instrument-common/src/main/scala/org/enso/interpreter/instrument/job/UpsertVisualizationJob.scala
@@ -239,7 +239,7 @@ object UpsertVisualizationJob {
         val (callback, arguments) = expression match {
           case Api.VisualizationExpression.Text(_, expression) =>
             val callback = ctx.executionService.evaluateExpression(
-              module,
+              expressionModule,
               expression
             )
             val arguments = Vector()
```